mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb

```diff
@@ -47,18 +55,34 @@ resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
             memory = "512Mi"
           }
         }
+        # [END cloudrun_service_configuration_limit_memory]
+        # [END cloudrun_service_configuration_limit_cpu]
+
+        # [START cloudrun_service_configuration_containers]
       }
+      # [END cloudrun_service_configuration_containers]
 
+      # [START cloudrun_service_configuration_timeout]
       # Timeout
       # https://cloud.google.com/run/docs/configuring/request-timeout
       timeout_seconds = 300
+      # [END cloudrun_service_configuration_timeout]
 
+      # [START cloudrun_service_configuration_concurrency]
       # Maximum concurrent requests
       # https://cloud.google.com/run/docs/configuring/concurrency
       container_concurrency = 80
+      # [END cloudrun_service_configuration_concurrency]
+
+      # [START cloudrun_service_configuration_containers]
     }
+    # [END cloudrun_service_configuration_containers]
 
+    # [START cloudrun_service_configuration_max_instances]
```

Comment:
This block id starts multiple times. Is that intentional.

See lines `81` and `110`

Reply:
Nope - fixed.